### ISSUE TITLE
Linki "dokup surowce" jak brakuje?

### ISSUE THREAD

**Issue by mstrojny** [mstrojny/cp2#15]
Potrzeba pomysłu jak, gdzie, czy warto, itp

**Comment by mstrojny** [comment]
- od darkherringa podobny pomysł padł
- w cpfanie?

**Comment by mstrojny** [comment]
"Kojarzę, ze kiedyś była taka możliwość, żeby automatycznie dokupić potrzebna ilość surowców do budowy/szkolenia/budowy statków" (chyba tweaks)

**Comment by Jacek-Siuda** [comment]
Tak, było to w CPTweaks. Niestety, widzę że w publicznych repo kod mi zeżarło :(

Jeśli dobrze pamiętam, to zapamiętywałem prędkość wydobycia i pod komunikatem o brakujących surowcach dawałem linka  do giełdy z dodatkowymi parametrami umożliwiającymi dokupienie surowca na który najdłużej trzebaby czekać. W giełdzie ustawiałem taką ilość, żeby w momencie dostarczenia surowca było go +dokładnie+ tyle ile trzeba.

**Comment by mstrojny** [comment]
Thx. To chyba nawet sprytniej niż potrzeba, bo giełda teraz szybsza, a i koszty są zmienne.
